### PR TITLE
P0 cart and wishlist

### PR DESCRIPTION
**Pull request by sumitk-singh**
waiting for the PR review of cart and Wishlist page

### CODE REVIEW

File: pages-css/cart-css/page-layout.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    font-family: 'Quicksand', sans-serif;
+    scroll-behavior: smooth;
+}
+
+
+/* page container */
+
+.page-layout-container {
+    display: grid;
+    gap: 0.313rem;
```
Comment (zoyatanweer) [inline pages-css/cart-css/page-layout.css]: convert this in pixels

File: pages-css/cart-css/page-layout.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: 0.313rem;
```
Comment (zoyatanweer) [inline pages-css/cart-css/page-layout.css]: write in px

File: pages-css/cart-css/page-layout.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    row-gap: 0.313rem;
+    grid-template-areas: "header header header " "section-1 section-1 section-1 " "section-2 section-2 section-3" "footer footer footer ";
+    padding: 0.313rem;
+    padding-left: 0.938rem;
+    padding-right: 0.938rem;
```
Comment (zoyatanweer) [inline pages-css/cart-css/page-layout.css]: write in pixels

File: pages-css/cart-css/page-layout.css [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    grid-area: header;
+    height: 3.75rem;
+}
+
+.section-1 {
+    grid-area: section-1;
+    padding-left: 1.563rem;
```
Comment (zoyatanweer) [inline pages-css/cart-css/page-layout.css]: i dont think you have to write upto 3 places of decimals. keep it limited to 1 max 2.

File: pages-css/cart-css/page-layout.css [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    padding-right: 1.563rem;
+}
+
+.section-2 {
+    grid-area: section-2;
+    padding-left: 1.563rem;
+    padding-right: 1.563rem;
```
Comment (zoyatanweer) [inline pages-css/cart-css/page-layout.css]: change here

File: pages-css/cart-css/page-layout.css (hunk 1 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: ok

File: pages-css/cart-css/page-layout.css [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+.section-3 {
+    grid-area: section-3;
+    padding-left: 1.563rem;
+    padding-right: 1.563rem;
```
Comment (zoyatanweer) [inline pages-css/cart-css/page-layout.css]: here also

File: pages-css/cart-css/page-layout.css (hunk 4 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: ok

File: pages-css/cart-css/page-layout.css (hunk 6 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: ok, let me fix it.

File: pages-css/wishlist-css/page-layout.css [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+    display: grid;
+    gap: 0.313rem;
+    grid-template-columns: 1fr 3fr 1fr;
+    column-gap: 0.313rem;
+    row-gap: 0.313rem;
+    grid-template-areas: "header header header " "section-1 section-1 section-1 " "section-2 section-2 section-2 " "footer footer footer ";
+    padding: 0.313rem;
+    padding-left: 0.938rem;
+    padding-right: 0.938rem;
+}
```
Comment (zoyatanweer) [inline pages-css/wishlist-css/page-layout.css:38]: change all these value upto 1 place of decimal. make it in pixels.

**Review by zoyatanweer — CHANGES_REQUESTED** [review]
do change the code in the places where I have mentioned.
Also you need not to add so many products into each category. There were many image files and so the PR became big, Try to keep it upto 500 or 600 lines.

good to merge with suggested changes.

File: pages-css/cart-css/page-layout.css [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+.footer {
+    background: var(--color-footer-background);
+    color: var(--color-white);
```
Comment (sudipkundu999) [inline pages-css/cart-css/page-layout.css]: the text is not readable change to a darker color

**Review by sudipkundu999 — APPROVED** [review]
As a rule of thumb when size is less than 1rem use px.
Try to keep PRs small.
Do the changes as suggested

File: pages-css/cart-css/page-layout.css (hunk 1 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: done

File: pages-css/cart-css/page-layout.css (hunk 2 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: done

File: pages-css/cart-css/page-layout.css (hunk 3 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: converted smaller value in px

File: pages-css/cart-css/page-layout.css (hunk 4 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: done

File: pages-css/cart-css/page-layout.css (hunk 5 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: done

File: pages-css/cart-css/page-layout.css (hunk 6 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: done

File: pages-css/wishlist-css/page-layout.css (hunk 7 above)
Comment (sumitk-singh) [reply pages-css/wishlist-css/page-layout.css:38]: value changed upto 1 place of decimal

File: pages-css/cart-css/page-layout.css (hunk 8 above)
Comment (sumitk-singh) [reply pages-css/cart-css/page-layout.css]: changed to dark color

**Review by zoyatanweer — APPROVED** [review]
good work, can merge now.

**Pull request merged**